azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -705,7 +705,8 @@ private static Props loadConfigurationFromAzkabanHome() {
 
   private void startWebMetrics() throws Exception {
 
-    final MetricRegistry registry = MetricsManager.INSTANCE.getRegistry();
+    final MetricRegistry registry = SERVICE_PROVIDER.getInstance(MetricsManager.class)
```

Comment:
That's a good point. I'll have it directly depend on both MetricsManager and MetricsRegistry then. Thanks again!